recipes/wetterdienst/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "wetterdienst" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c6be18b0cd82f634d385e8168193a8a6f42d9a24e66ac171fe33e07180aa3031
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
Use grayskull for creating the recipe and you'll get the latest "script" line here.

Reply:
@ocefpaf Thanks! Will try this next time.